### DISCUSSION TITLE
Welcome to FluentUI Discussions!

### DISCUSSION THREAD

**Discussion by zhuzichu520** [zhuzichu520/FluentUI#95] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by nrbnlulu** [comment]
Hey, it would be nice if the repo was in english

**Comment by parker-int64** [comment]
Add English support.
我先尝试翻一下，然后交pr看看
